docker/Dockerfile

```diff
@@ -93,8 +90,11 @@ RUN \
            /usr/local/lib/libgtest* \
            /usr/local/share/aclocal/gtest* \
            /opt/fiware-orion \
+           /usr/share/cracklib \
```

Comment:
It is safe to directly remove directories from /usr/share? Shouldn't they be removed using "yum remove XXX"?